src/imitation/scripts/config/train_preference_comparisons.py

```diff
@@ -48,6 +55,7 @@ def train_defaults():
     allow_variable_horizon = False
 
     checkpoint_interval = 0  # Num epochs between saving (<0 disables, =0 final only)
+    query_schedule = "hyperbolic"
```

Comment:
Reasonable default given it's what's used in paper, but worth noting this is changing the default behavior (previously we were effectively constant). I think that's fine but @yawen-d @levmckinney you should be aware of this -- you may want to override it in your experiments if you want to make direct comparisons.

Reply:
Thanks for the heads up!